wandb/wandb_torch.py

```diff
@@ -410,6 +412,9 @@ def hook_torch_modules(
                 wandb.run.history.torch._hook_handles[
                     "topology/" + str(id(graph_hook))
                 ] = graph_hook
+                if not hasattr(parent, "_wandb_hook_names"):
+                    # should never happen but let's be extra safe
+                    parent._wandb_hook_names = []
```

Comment:
I don't actually see a bug that was fixed anywhere in this PR.  How did the user get the exception when they disabled watch?

Reply:
This section should not be needed but it just makes it extra safe in case there is a type of model I didn't think of.